doc/_extensions/odoo_ext/sub-menu_list.html

```diff
@@ -1,8 +1,9 @@
 {# note: if odoo.com/page/docs sub-menu's structure change, these links have to change as well #}
 
-<li><a href="http://www.odoo.com/documentation/user/10.0/index.html">User</a></li>
-<li><a href="http://www.odoo.com/documentation/10.0/index.html">Developer</a></li>
-<li><a href="http://www.odoo.com/documentation/10.0/api_integration.html">API</a></li>
```

Comment:
that one is now at https://www.odoo.com/documentation/11.0/webservices/odoo.html for 11.0